rclone_python/rclone.py

```diff
@@ -262,6 +262,39 @@ def delete(path: str, args=None):
             f'Deleting path "{path}" failed with error message:\n{process.stderr}'
         )
 
+@__check_installed
+def link(
+    path: str,
+    expire: Union[str, None] = None,
+    unlink=False,
+    args=None,
+) -> str:
+    """
+    Generate public link to file/directory.
+    :param path: The path to the file/directory that should be create, retrieve or remove a public link.
+    :param expire: The amount of time that the link will be valid
+    :param unlink: If true, remove existing public links to the file or directory.
+    :param args: List of additional arguments/ flags.
+    :return: The link to the given file or directory.
+    """
```

Comment:
```suggestion
    """
    Generates a public link to a file/directory.
    :param path: The path to the file/directory that a public link should be created, retrieved or removed for.
    :param expire: The amount of time that the link will be valid
    :param unlink: If true, remove existing public links to the file or directory.
    :param args: List of additional arguments/ flags.
    :return: The link to the given file or directory.
    """
```

Reply:
Thanks for you contribution! 
I think it makes sense to add the `not supported by all backends` hint to the two parameters, as apparently quite a few backends do not support these options.